### PR TITLE
Add SAML documentation

### PR DESCRIPTION
**Pull request by rdenarie**

### CODE REVIEW

File: docs/administration/saml-integration.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   identifying the user. The form contains SAMLResponse value.
+5. The browser requests assertion consumer service at the SP.
+6. The consumer service processes the SAMLResponse, creates a security
+   context and redirects to the target resource.
+7. The browser requests target resource again.
+8. The SP finds a security context, so it returns the target resource.
+
+![image0](/img/administration/saml/salm-sequence.png)
+
+In this document, two addresses, www.sp.com and www.idp.com, and folders
```
Comment (ahamdi) [inline docs/administration/saml-integration.md]: We manage just the case where eXo platform is a SP, could you update this accordingly ?

File: docs/administration/saml-integration.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+## Generating and using your own keystore
+
+The default `jbid_test_keystore.jks` is useful for testing purpose,
+but in eXo Platform you need to generate and use your own keystore as
+follows:
+
+1. Generate your file using the **keytool** command:
+   
+```bash
+keytool -genkey -alias secure-key -keyalg RSA -keystore secure-keystore.jks
```
Comment (ahamdi) [inline docs/administration/saml-integration.md:113]: Is there some requirements for how to generate the certificate and the keystore ? especially for the common name attribute ?

Comment (rdenarie) [reply docs/administration/saml-integration.md:113]: I dont know exactly what are the requirement, and I think this part is not really our expertise, so I prefer to not add more details on that

File: docs/administration/saml-integration.md (hunk 1 above)
Comment (rdenarie) [reply docs/administration/saml-integration.md]: done

**Review by ahamdi — APPROVED** [review]
LGTM

**Pull request merged**